at_commons/lib/src/verb/sync_verb_builder.dart

```diff
@@ -5,15 +5,20 @@ class SyncVerbBuilder implements VerbBuilder {
 
   String? regex;
 
-  bool isStream = false;
+  int limit = 10;
```

Comment:
this limi should be in at_client..because if we have to change this..then at_commons version has to be incremented and dependent packages changed